<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Task Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>671</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="TableView_Activity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>651</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>56</height>
      </rect>
     </property>
     <property name="title">
      <string>Activities</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Button_NewActivity">
        <property name="text">
         <string>New Activity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_EditActivity">
        <property name="text">
         <string>Edit Activity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_DeleteActivity">
        <property name="text">
         <string>Delete Activity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_ShowAll">
        <property name="text">
         <string>Show All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>671</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="TableView_Tasks">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>61</y>
       <width>651</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>56</height>
      </rect>
     </property>
     <property name="title">
      <string>Tasks</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Button_UpdateTask">
        <property name="text">
         <string>Update Task</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_RemoveTask">
        <property name="text">
         <string>Remove Task</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>451</width>
       <height>56</height>
      </rect>
     </property>
     <property name="title">
      <string>Filter the Task</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="RadioButton_AllTasks">
        <property name="text">
         <string>All Tasks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioButton_Finished">
        <property name="text">
         <string>Finished</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioButton_OnGoing">
        <property name="text">
         <string>On Going</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioButton_NotStarted">
        <property name="text">
         <string>Not Started</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineEdit_FilterTask">
        <property name="placeholderText">
         <string>Filter by part of text...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Button_NewActivity</tabstop>
  <tabstop>Button_EditActivity</tabstop>
  <tabstop>Button_DeleteActivity</tabstop>
  <tabstop>Button_ShowAll</tabstop>
  <tabstop>Button_UpdateTask</tabstop>
  <tabstop>Button_RemoveTask</tabstop>
  <tabstop>RadioButton_AllTasks</tabstop>
  <tabstop>RadioButton_Finished</tabstop>
  <tabstop>RadioButton_OnGoing</tabstop>
  <tabstop>RadioButton_NotStarted</tabstop>
  <tabstop>LineEdit_FilterTask</tabstop>
  <tabstop>TableView_Activity</tabstop>
  <tabstop>TableView_Tasks</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
